angular-workspace/projects/ni/nimble-angular/table-column/date-text/nimble-table-column-date-text.directive.ts

```diff
@@ -0,0 +1,301 @@
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customTimeZone', value);
+    }
+
+    public get customCalendar(): string | undefined {
+        return this.elementRef.nativeElement.customCalendar;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-calendar') public set customCalendar(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customCalendar', value);
+    }
+
+    public get customDayPeriod(): DayPeriodFormat {
+        return this.elementRef.nativeElement.customDayPeriod;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-day-period') public set customDayPeriod(value: DayPeriodFormat) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customDayPeriod', value);
+    }
+
+    public get customNumberingSystem(): string | undefined {
+        return this.elementRef.nativeElement.customNumberingSystem;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-numbering-system') public set customNumberingSystem(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customNumberingSystem', value);
+    }
+
+    public get customDateStyle(): DateStyle {
+        return this.elementRef.nativeElement.customDateStyle;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-date-style') public set customDateStyle(value: DateStyle) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customDateStyle', value);
+    }
+
+    public get customTimeStyle(): TimeStyle {
+        return this.elementRef.nativeElement.customTimeStyle;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-time-style') public set customTimeStyle(value: TimeStyle) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customTimeStyle', value);
+    }
+
+    public get customHourCycle(): HourCycleFormat {
+        return this.elementRef.nativeElement.customHourCycle;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('custom-hour-cycle') public set customHourCycle(value: HourCycleFormat) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'customHourCycle', value);
+    }
+
+    public get fractionalWidth(): number | null | undefined {
```

Comment:
Man...we need to get those [mixins](https://github.com/ni/nimble/issues/1211) created.